core/common.py

```diff
@@ -7,6 +10,8 @@ def date_is_correct(date):
     for date_item in date_items:
         if not date_item.isdigit():
             return False
+    if len(date_items[1]) != 2 or len(date_items[2]) != 2:
```

Comment:
```suggestion
    if list(map(len, date_items[1:2])) != [2, 2]
```

Reply:
There should be [1:3], but I will use this